client/look-and-feel/css/src/Stepper/Stepper.stories.ts

```diff
@@ -12,8 +12,10 @@ export const Stepper: StoryObj = {
     const stepper = document.createElement("div");
     stepper.innerHTML = `
     <div class="af-stepper-client">
-      <h2 class="af-stepper-client__title" aria-describedby=":r4:">Step 1</h2>
-      <p class="af-stepper-client__subtitle">Step subtitle</p>
+      <div class="af-stepper-client__header">
```

Comment:
la classe `af-stepper-client__header` existe pas dans le css elle est utile cette div ?

Reply:
En effet aucun style n'est appliqué dessus, je l'ai surtout mis pour qu'elle soit accessible et que si un projet a besoin d'appliquer du style sur ce container, alors il pourra

Reply:
Mais si tu préfères je l'enlève et le jour où quelqu'un aura besoin d'override le style du container alors il le rajoutera.

Reply:
je pense pas que ça soit utile, si t'utilises le designsystem t'auras plus tendance à ajouter une classname sur le composant stepper et surcharger côté css en faisant par exemple  `.ta_class > h2 { ... }`
en regardant le code tu sais pas que la classe header est là pour être surchargée

Reply:
Fixed in : https://github.com/AxaFrance/design-system/pull/894/commits/a0638408913cca5f3fe19480ba4ebc8affa4f7f6